src/modules/launcher/Wox/ViewModel/ResultsViewModel.cs

```diff
@@ -192,6 +192,7 @@ public void AddResults(List<Result> newRawResults, string resultId)
                 else
                 {
                     Margin = new Thickness { Top = 0 };
+                    Visbility = Visibility.Collapsed;
```

Comment:
Quick question, why not Visibility.Hidden?

Reply:
@alekhyareddy28 The difference is that Visibility.Hidden hides the control but reserves the space it occupies in the layout. So it renders whitespace instead of the control. Visibilty.Collapsed does not render the control and does not reserve the whitespace. In our case either of these would work. And I tried out both but I didn't see any visible performance improvement. Also since we have been using Visibility.Collapsed in other parts of code, I let it be collapsed to match it.

Reply:
Sounds good! I just asked with the performance in mind as I remember you mentioning in a previous comment that Visibility.Hidden was faster.